model/src/form/form-metadata/types.ts

```diff
@@ -84,12 +84,39 @@ export interface FormMetadata {
    * The live state of the form
    */
   live?: FormMetadataState
+
+  /**
+   * The author who created the form
+   */
+  createdBy: FormMetadataState['createdBy']
+
+  /**
+   * The date the form was created
+   */
+  createdAt: FormMetadataState['createdAt']
+
+  /**
+   * The author who last updated the form
+   */
+  updatedBy: FormMetadataState['updatedBy']
+
+  /**
+   * The date the form was last updated
+   */
+  updatedAt: FormMetadataState['updatedAt']
 }
 
 export type FormByIdInput = Pick<FormMetadata, 'id'>
 export type FormBySlugInput = Pick<FormMetadata, 'slug'>
 export type FormMetadataDocument = Omit<FormMetadata, 'id'>
 export type FormMetadataInput = Omit<
   FormMetadata,
-  'id' | 'slug' | 'draft' | 'live'
+  | 'id'
+  | 'slug'
+  | 'draft'
+  | 'live'
+  | 'createdBy'
+  | 'createdAt'
+  | 'updatedBy'
+  | 'updatedAt'
```

Comment:
If you find this a bit much, you can use `Pick<FormMetadata, 'one' | 'two'>` instead?

i.e. Pick what's allowed versus remembering to exclude what's not

Reply:
That works too! Updated.